tests/scheduler/test_scheduler.py

```
@@ -339,11 +339,73 @@ def test_check_manifest_pass(basic_project):
+               autospec=True) as check_manifest, \
+            patch("siliconcompiler.scheduler.Scheduler._Scheduler__run_setup") as run_setup, \
+            patch("siliconcompiler.scheduler.Scheduler.configure_nodes") as configure_nodes, \
+            patch("siliconcompiler.scheduler.Scheduler._Scheduler__check_tool_requirements") \
+            as check_tool_requirements, \
+            patch("siliconcompiler.scheduler.Scheduler._Scheduler__clean_build_dir_incr") \
+            as clean_build_dir_incr, \
+            patch("siliconcompiler.scheduler.Scheduler._Scheduler__check_flowgraph_io") \
+            as check_flowgraph_io:
+        check_manifest.return_value = True
+        check_tool_requirements.return_value = False
+        with pytest.raises(RuntimeError, match='^Tools requirements not met$'):
```

Ruff (0.13.3)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)